packages/datepicker/src/react/utils.ts

```diff
@@ -35,32 +25,34 @@ export const useIsInRange = (selected: Date[] = []) => {
     setHoveredDate(date)
   }
 
-  const isInRange = (date: Date) => {
+  const isInRange = (date: Date): string => {
     if (selected.length) {
       const firstSelected = selected[0].getTime()
       if (selected.length === 2) {
         const secondSelected = selected[1].getTime()
-        return {
-          ...style.inRange(
-            firstSelected < date.getTime() && secondSelected > date.getTime()
-          ),
-          ...(firstSelected === date.getTime() && style.start()),
-          ...(secondSelected === date.getTime() && style.end())
-        }
+        return classNames(
```

Comment:
This hook returning a specific attribute (className) that other props needs merged into has implications for usage. See story for the merge required.